docs/docs/state.md

```diff
@@ -45,37 +43,39 @@ class Demo extends Component {
 
 ## State flows
 
-The most typical flow of the state is when the gesture handler first picks up the initial touch events, then at some point it recognizes the touches and after the end of the gesture is recognized it resets back to the initial state. The flow looks as follows (longer arrows represent that there are possibly more touch events received before the state changes):
+The most typical flow of state is when a gesture handler picks up on an initial touch event then recognizes it then acknowledges its ending then resets itself back to the initial state. 
+
+The flow looks as follows (longer arrows represent that there are possibly more touch events received before the state changes):
 
 [`UNDETERMINED`](#undetermined) -> [`BEGAN`](#began) ------> [`ACTIVE`](#active) ------> [`END`](#end) -> [`UNDETERMINED`](#undetermined)
 
-Here is another possible flow when the handler receive touches that causes it to fail recognition:
+Another possible flow is when a handler receives touches that cause a recognition failure:
 
 [`UNDETERMINED`](#undetermined) -> [`BEGAN`](#began) ------> [`FAILED`](#failed) -> [`UNDETERMINED`](#undetermined)
 
-Lastly even when the handler properly recognizes the gesture it may be interrupted by the touch system, in that case the flow looks as follows:
+Another possible flow is when a handler *does* properly recognize the gesture but then is interrupted by the touch system causing it to cancel. In that case the flow looks as follows:
 
 [`UNDETERMINED`](#undetermined) -> [`BEGAN`](#began) ------> [`ACTIVE`](#active) ------> [`CANCELLED`](#cancelled) -> [`UNDETERMINED`](#undetermined)
 
 ## States
 
-Section below lists all the possible handler states along with detailed description of each state:
+The section below describes all possible handler states:
 
 ### UNDETERMINED
-This is the initial state of each handler. It also changes to that state when it is done recognizing and hasn't started recogninzing another gesture.
+This is the initial state of each handler. It can be found before gestures have occurred and after they've completed. 
```

Comment:
```suggestion
This is the initial state of each handler and it goes into this state after it's done with recognizing a gesture.
```

For me, "before gestures have occurred and after they've completed" isn't precise enough, we want to signify the end of recognizing a gesture.

Reply:
just a quick change to -> "after it's done recognizing a gesture"